AsyncDisplayKit/ASCellNode.h

```diff
@@ -108,6 +108,14 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
  */
 @property (nonatomic, assign, getter=isHighlighted) BOOL highlighted;
 
+/**
+ * The current index path of this cell node, or @c nil if this node is
+ * not a valid item inside a table node or collection node.
+ *
+ * @note This property must be accessed on the main thread.
+ */
+@property (nonatomic, strong, readonly, nullable) NSIndexPath *indexPath;
```

Comment:
No need for `strong` here. Computed properties having no storage qualifier is a pattern e.g. `UICollectionView.visibleCells`.